src/components/Auth/api.ts

```diff
@@ -0,0 +1,65 @@
+type MethodTypes = 'GET' | 'POST' | 'PUT' | 'DELETE'
```

Comment:
@selankon shoulnd't this file be in a different folder? Maybe the queries folder? or another utils folder?